tests/conftest.py

```diff
@@ -2,16 +2,25 @@
 
 import contextlib
 import os
-from collections.abc import Iterator
+from collections.abc import Generator, Iterator
 from pathlib import Path
 from shutil import copytree
 from tempfile import TemporaryDirectory
 from unittest.mock import patch
 
 import pytest
 
+from lsst.rsp import _discovery
 from lsst.rsp.startup.storage.command import Command
 
+
```

Comment:
Yeah here's where I would have defined `DISCOVERY_PATH` in constants and overridden that, but it doesn't matter.